_stbt/core.py

```diff
@@ -2628,14 +2628,15 @@ def _hocr_iterate(hocr):
 
 
 def _hocr_find_phrase(hocr, phrase):
-    l = list(_hocr_iterate(hocr))
-    words_only = [(w, elem) for w, elem in l if w.strip() != u'']
+    words_only = [(w.lower().translate(_ocr_transtab), elem)
+                  for w, elem in _hocr_iterate(hocr) if w.strip() != u'']
+    phrase = [unicode(w).lower().translate(_ocr_transtab) for w in phrase]
```

Comment:
If it's a byte string, the call to `translate(_ocr_transtab)` would fail with:

    TypeError: expected a string or other character buffer object

I need to look into what `translate` is actually doing -- maybe we'll need an explicit type check and 2 different code paths.